backend/src/app/http/debug.clj

```diff
@@ -430,6 +430,50 @@
      ::yres/body    "OK"}))
 
 
+(defn- add-team-feature
+  [{:keys [params] :as request}]
+  (let [team-id (some-> params :team-id d/parse-uuid)
+        feature (some-> params :feature d/read-string)
```

Comment:
I think we should not use d/read-string, because this interprets it as clojure data structure:

```clojure
user=> (d/read-string "foo/bar")
foo/bar
user=> (type (d/read-string "foo/bar"))
clojure.lang.Symbol
```
So here we just need to take the value as is.

Reply:
Got it!! I think that's the reason why the disjoint didn't work on the `disable-team-feature` and I had to cast it there, let me fix that!